src/compiler/style/optimize-css.ts

```diff
@@ -8,6 +8,8 @@ export const optimizeCss = async (
   compilerCtx: d.CompilerCtx,
   diagnostics: d.Diagnostic[],
   styleText: string,
+  // TODO(STENCIL-1076): Investigate removing this parameter, which appears to be unused. This function is exported by
+  // the compiler, making this a breaking change should be remove it.
```

Comment:
```suggestion
  // the compiler, making this a breaking change should we remove it.
```

Reply:
Ah, good catch, thanks! (Done, and squashed the change since this was non-blocking)